index.js

```diff
@@ -19,26 +19,48 @@ const manageRecipes = async () => {
     await Recipe.deleteMany();
 
     // Run your code here, after you have insured that the connection was made
+
+    
+    const myRecipe = {
+      title: "Asian Glazed Chicken Thighs",
```

Comment:
You need to change this recipie name otherwise on the next iteration where you're going to add all the recipies you get an error saying "duplicate key on title"